File: FASE 1 - Variáveis e Tipos de Dados/Desafio 2/calcularSalario.cs

//GPTando - Mini-Missões - Fase 1 - Missão 2

using System;
using System.Globalization; //necessário para o CultureInfo reconhecer a moeda do país

class CadastroSimples{
    static void Main(){

        Console.Write ("Digite seu salário mensal: ");
        string input = Console.ReadLine ();
        input = input.Replace (",", ".");
        double salarioMensal = double.Parse (input);

        Console.Write ("Digite quantidade de meses trabalhados: ");
        int mesesTrabalhados = int.Parse (Console.ReadLine ());

        double salarioTotal = salarioMensal * mesesTrabalhados;

        //Console.Write ("Seu salário total, nesses " + mesesTrabalhados + " meses, é de " + salarioTotal + ".");
        
        //pode adicionar ToString para dar o modelo de moeda
        Console.Write ("Seu salário total, nesses " + mesesTrabalhados + " meses, é de " + salarioTotal.ToString("C", new CultureInfo ("pt-BR")) + ".");

        //Outra opção é usar interpolação:
        //Console.Write ($"Seu salário total, nesses {mesesTrabalhados} meses, é de {salarioTotal}.");
        //assim como em JavaScript, adicionar $ e trocar as aspas das strings e "+", por apenas uma apas dupla na abertura e no fechamento
        
        //ou adicionar C na interpolação para dar o modelo moeda
        //Console.Write ($"Seu salário total, nesses {mesesTrabalhados} meses, é de {salarioTotal("C", new CultureInfo ("pt-BR"))}.");


    }

}

File: FASE 1 - Variáveis e Tipos de Dados/Desafio 2/calcularSalario.js

//GPTando - Mini-Missões - Fase 1 - Missão 2


let salarioMensal = parseFloat(prompt("Digite seu salário mensal: ").replace(",", ".")); //"replace" vai fazer com que mesmo que o usuário escreva a vírgula, seja lido como ponto. 
let mesesTrabalhados = parseInt(prompt("Digite quantidade de meses trabalhado: "));
let salarioTotal = salarioMensal * mesesTrabalhados;

console.log ("Seu salário total, nesses " , mesesTrabalhados, " meses, é de " , salarioTotal, ".");

//Outra opção é colocar o formatador para transformar o "salarioTotal" em moeda, mas também trocando a virgula que separa as variáveis e as strings por "+"
console.log ("Seu salário total, nesses " + mesesTrabalhados + " meses, é de " 
    //e ao adicionar o seguinte código, transforma "salarioTotal" para moeda
    +new Intl.NumberFormat("pt-BR", {style: "currency", currency: "BRL"}).format(salarioTotal) + ".");

//ou trocar todas as aspas duplas da String por uma crase de abertura e fechamento, que faz com que a variável possa se adicionada à ${ }, sem trocar as virgulas.  OBS.: PODEM SER DESCONSIDERADAS AS VIRGULAS QUE SEPARAM OS ELEMENTOS DO CONSOLE, DEIXANDO SÓ AS DA STRING
console.log (`Seu salário total, nesses ${mesesTrabalhados}  meses, é de  
     ${new Intl.NumberFormat("pt-BR", {style: "currency", currency: "BRL"}).format(salarioTotal)}.`);